frontend/public/extend/devconsole/components/topology/shapes/KnativeDecorator.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-unused-vars, no-undef */
+import * as React from 'react';
+import { getImageForIconClass } from '../../../../../components/catalog/catalog-item-icon';
+
+type KnativeDecoratorTypes = {
+  x: number;
+  y: number;
+  width: number;
+  height: number;
+};
+
+const KnativeDecorator: React.FunctionComponent<KnativeDecoratorTypes> = ({
```

Comment:
This shouldn't be a decorator. Rename accordingly:
```suggestion
const KnativeIcon: React.FC<KnativeIconProps> = ({
```